docs/new_problem.md

```diff
@@ -240,16 +240,38 @@ All hyperparamters inherit from `_default_hparams()` in `problem.py.` If you wou
 from tensor2tensor.models import transformer
 
 @registry.register_hparams
-def word2def_hparams(self):
+def word2def_hparams():
     hparams = transformer.transformer_base_single_gpu()  # Or whatever you'd like to build off.
     hparams.batch_size = 1024
     return hparams
 ```
 
+# Test the data generation
+If you are using a local checkout of tensor2tensor, you can install the local version in developer mode with `pip installe -e .` from the tensor2tensor directory.
+
+You will also need to add your new file to the [`all_problems.py`](https://github.com/tensorflow/tensor2tensor/blob/master/tensor2tensor/data_generators/all_problems.py) file too.
```

Comment:
Again, it should be clear that this is needed only at the moment when I want to contribute my dataset/problem to t2t. If I want to keep it private (e.g. because the resources are not publicly&automatically downloadable), the usr_dir solution is preferred.

Reply:
done